app/questions/services/mailchimp.py

```diff
@@ -66,14 +67,19 @@ def send_email(clients, list_id, workflow_id, email_id):
         person = {
             "email_address": email,
             "status": "subscribed",
-            "merge_fields": {"SUB_ID": submission.id},
+            "merge_fields": {"SUB_ID": str(submission.id)},
         }
-        # Add person to list
-        mailchimp.lists.members.create(list_id=list_id, data=person)
-        # Send person email
-        mailchimp.automations.emails.queues.create(
-            workflow_id=workflow_id, email_id=email_id, data=person
-        )
+
+        try:
+            # Add person to list and send them email
+            mailchimp.lists.members.create(list_id=list_id, data=person)
+            mailchimp.automations.emails.queues.create(
+                workflow_id=workflow_id, email_id=email_id, data=person
+            )
+        except MailChimpError:
+            # Skip if their email is already on list
+            continue
```

Comment:
this should be "pass" not "continue"
please log this exception with logger.exception() so we know when this happens

Reply:
eg. if this exception is getting thrown 1000 times a day and swallowed by this except block, that would be bad

Reply:
otherwise good work identifying this issue and proposing a fix!

Reply:
Surprisingly, according to the Python docs, "the pass statement does nothing. It can be used when a statement is required syntactically but the program requires no action." 

Logging message will give you the ID of the submission, and the traceback (with email address inside).

Reply:
oh I see it's in a loop, I missed that